js/popover.js

```diff
@@ -65,10 +65,16 @@
     var $e = this.$element
     var o  = this.options
 
-    return $e.attr('data-content')
+    var content = $e.attr('data-content')
       || (typeof o.content == 'function' ?
             o.content.call($e[0]) :
             o.content)
+
+    if (['[object Function]', '[object Object]', '[object String]'].indexOf(Object.prototype.toString.call(content)) === -1) {
```

Comment:
Am I missing something or why isn't something like `if (/string|function/.test(typeof content))` used here? Also, according to the docs a jQuery object isn't even allowed for the `content` option.